index.html

```diff
@@ -142,6 +142,18 @@ <h5>Languages</h5><img src="./images/Union.png" alt="expand" class="arrow"></li>
           <li class="expertise"><h5>Skills</h5><img src="./images/Union2.png" alt="expand" class="arrow"></li>
         </ul>
       </section>
+
+      <section id="form-section">
+        <h2>contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built ora project that needs coding. I’d love to help with it </p>
+
+        <form id="form" action="https://formspree.io/f/xjvlpwvy" method="post">
+          <input id="name" type="text" maxlength="30" value="Yeremias">
+          <input id="email" type="email" value="natajayanj14@gmail.com">
+          <textarea id="text" maxlength="500" placeholder="Enter text here"></textarea>
+          <button type="submit">Get It Touch</button>
+        </form>
+      </section>
```

Comment:
- [ ] I get an error when I try to submit a form with inputs. Please kindly fix the error

Reply:
Hi Anselem, thanks for your review.

When I fill out the form, I don't get errors, and I receive an email telling me it was successfully submitted.

<img width="367" alt="Screen Shot 2022-04-27 at 5 00 21 PM" src="https://user-images.githubusercontent.com/33801345/165536134-fe04b769-5b17-499b-ab1c-83c425062ac0.png">

### Would you post the error you received here, please?